devel/code_generation.py

```diff
@@ -13,8 +14,11 @@ def main():
 
     output_fnames = find_files_with_templates('.')
     print(f'Found {len(output_fnames)} files with templates')
+    print(dict(extensions=extensions))
     for output_fname in output_fnames:
         process_template(output_fname, template_kwargs=dict(extensions=extensions))
+    
+    create_md_ts_files('.')
```

Comment:
# For every .md file that also has a .md.gen.ts file, generate the .md.gen.ts file so that the markdown can be imported via `import x from './---.md.gen'`
# The previous raw.macro solution did not work with jupyter extension